docs/develop/teams-toolkit-overview.md

```diff
@@ -0,0 +1,48 @@
+description: Learn how to create Office Add-in projects using the Teams Toolkit.
+ms.date: 02/27/2023
+ms.localizationpriority: high
+---
+
+# Create Office Add-in projects with Teams Toolkit (preview)
+
+Extensions to the Microsoft 365 Platform are called "Teams Apps", even if the extension is for a part of Microsoft 365 outside of the Teams application itself. A primary tool for developing Teams Apps is the Teams Toolkit. You can create Office Add-ins with the Teams Toolkit subject to the following restrictions:
+
+- Only Outlook Add-ins can be created at this time. We're working hard to enable support in Teams Toolkit for add-ins to other Office applications.
+- Add-ins created with Teams Toolkit use the unified Microsoft 365 manifest which is currently in preview. We encourage you to experiment with creating add-ins using the toolkit but don't use the unified manifest for production add-ins.
+
+Install the prerelease version of Teams Toolkit into Visual Studio Code as described in [Install Teams Toolkit - Prerelease version](/microsoftteams/platform/toolkit/install-teams-toolkit?tabs=vscode&pivots=visual-studio-code#install-a-pre-release-version).
+
+## Create an Outlook Add-in project
+
+1. Open Visual Studio Code and select the Teams Toolkit icon in the **Activity Bar**.
+
+    ![Screenshot showing the Teams Toolkit icon.](../images/teams-toolkit-icon.png)
+
+1. Select **Create a new app**.
+1. In the **Select an option** drop down, select **Start with an Outlook add-in**.
+
+    ![Screenshot showing three options in Teams Toolkit option drop down. The second option "Start with an Outlook add-in" is selected.](../images/teams-toolkit-create-outlook-add-in.png)
+
+1. In the **Select a capability** drop down, select **Outlook Taskpane Add-in (preview)**.
+
+    ![Screenshot showing two options in Teams Toolkit capability drop down. The first option "Outlook Taskpane Add-in (preview)" is selected.](../images/teams-toolkit-create-outlook-task-pane-capability.png)
+
+1. In the **Workspace folder** dialog that opens, select the folder where you want to create the project.
+1. Give a name to the project (with no spaces) when prompted. Teams Toolkit will create the project with basic files and scaffolding. It will then open the project *in a second Visual Studio Code window*. Close the original Visual Studio Code window.
+1. In the Visual Studio Code **TERMINAL** navigate to the root of the project and run `npm install`.
+1. Before you make changes to the project, verify that Outlook add-in sideloading from Visual Studio Code works. Use the following steps:
+
+    <ol type="a">
+      <li>Ensure that your account in your Microsoft 365 developer tenancy is also an email account in desktop Outlook. If it isn't, follow the guidance in <a href="https://support.microsoft.com/office/add-an-email-account-to-outlook-e9da47c4-9b89-4b49-b945-a204aeea6726">Add an email account to Outlook</a>.</li>
+      <li><i>Close Outlook desktop.</i></li>
```

Comment:
Why is this italicized?

Reply:
It's something that is easy to overlook and really important, but I'll remove the italics.

Reply:
Maybe bold it then? Something about the whole line being italicized makes it seem _less_ important. Almost like it's a footnote.